vault/resource_quota_lease_count_test.go

```diff
@@ -0,0 +1,89 @@
+package vault
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+	"github.com/hashicorp/vault/api"
+)
+
+func randomQuotaLeaseString() string {
+	whole := float64(acctest.RandIntRange(1000, 2000))
+	decimal := float64(acctest.RandIntRange(0, 100)) / 100
+
+	leaseCout := fmt.Sprintf("%.1f", whole+decimal)
+	// Vault retuns floats with trailing zeros trimmed
+	return strings.TrimRight(strings.TrimRight(leaseCout, "0"), ".")
+}
```

Comment:
I think this should be generating an integer string instead of a float.

Reply:
Updated, as `float` this fails the test